include/crocoddyl/multibody/wrench-cone.hxx

```diff
@@ -299,6 +302,7 @@ void WrenchConeTpl<Scalar>::set_box(const Vector2s& box) {
 
 template <typename Scalar>
 void WrenchConeTpl<Scalar>::set_mu(const Scalar mu) {
+  mu_ = mu;
```

Comment:
Nice catch! Thanks for fixing this.

This also means we're not properly unit-testing this. I would define a `test_setter` function in https://github.com/loco-3d/crocoddyl/blob/master/unittest/test_wrench_cone.cpp